renamer.js

```diff
@@ -163,7 +169,7 @@ function renameFiles(files) {
 
   _.each(files, function(file) {
     if (file.new) {
-      fs.rename(file.original, file.new, function(err) {
+      fs.rename(process.argv[2] + '/' + file.original, process.argv[2] + '/' + file.new, function(err) {
```

Comment:
Please use `path.join()` for path concatenation so that the script remains portable (needs also a `path = require('path')` at the top)